giveOutput.js

```diff
@@ -0,0 +1,21 @@
+const readlineSync = require('readline-sync');
+
+function decideOutput(accounts) {
+    let command = readlineSync.question('List All or List [name]? ');
+
+    if (command === 'List All') {
+        for (let i = 0; i < accounts.length; i++) {
+            delete accounts[i].Transactions;

```

Comment:
It works here, but this deletion is a little scary overall; if you needed the accounts after running this the transactions would all be gone!

It'd be preferable to generate + log a nice string for each account.